docs/images.md

```diff
@@ -56,6 +56,13 @@ The [gridsome-transformer-remark](/plugins/transformer-remark) transformer plugi
 ![Alternative text](./image.jpg)
 ```
 
+## Usage in CSS
+Referencing [#411](https://github.com/gridsome/gridsome/issues/411), image that needs to be called via CSS should be called like the example below ( please note the `~@` ):
+
+```css
+background-image: url(~@/path/to/image.png)
```

Comment:
I ran a quick test on this and I believe the `~@/path/to/image.png` should be wrapped in quotes.